backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraSingleton.java

```diff
@@ -30,7 +30,7 @@
         void run() throws Exception;
     }
 
-    private static final int MAX_TEST_PLAYED = 100;
+    private static final int MAX_TEST_PLAYED = 200;
```

Comment:
the commit should not repeat what we can read (the how) but explain (the why)

Reply:
Because things don't break when I double this value, yet testing time is divided by nearly two.

Reply:
please put that into the comment